.github/workflows/node.js.yml

```
@@ -29,3 +29,51 @@
+      - name: Run ESLint
+        run: npm run lint
+
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    needs: [build, lint]
+    if: github.ref == 'refs/heads/main'
+
+    steps:
+      - name: Download artifact from build job
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)